tiledb/sm/subarray/subarray.cc

```diff
@@ -1204,7 +1204,7 @@ Status Subarray::compute_relevant_fragment_est_result_sizes(
   auto all_dims_fixed = array_schema->domain()->all_dims_fixed();
   auto num_threads = compute_tp->concurrency_level();
   auto ranges_per_thread = (uint64_t)ceil((double)range_num / num_threads);
-  auto statuses = parallel_for(compute_tp, 0, num_threads, [&](uint64_t t) {
+  auto status = parallel_for(compute_tp, 0, num_threads, [&](uint64_t t) {
```

Comment:
Did you mean to commit this file?

Reply:
Yeah, this was intentional. The `parallel_for*` routines previously returned a `vector<Status>` to support an interface from TBB. With those paths removed, this return value is simplified to a single `Status`.

Reply:
Thanks I missed the API change